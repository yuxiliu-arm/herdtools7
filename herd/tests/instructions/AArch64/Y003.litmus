AArch64 Y003
(* MUL *)

{
    0: X1 = 2;
    0: X2 = 3;
}

P0 ;
MUL W3, W1, W2;

forall (
    0: X1 = 2 /\
    0: X2 = 3 /\
    0: X3 = 6
)
