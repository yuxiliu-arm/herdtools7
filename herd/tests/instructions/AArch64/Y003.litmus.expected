Test Y003 Required
States 1
0:X1=2; 0:X2=3; 0:X3=6;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:X1=2 /\ 0:X2=3 /\ 0:X3=6)
Observation Y003 Always 1 0
Hash=7c6f007938f27a483b5b6ffa4f06c270

